addons/l10n_my_ubl_pint/models/account_edi_xml_pint_my.py

```diff
@@ -0,0 +1,96 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+from odoo import models, _
+
+
+class AccountEdiXmlUBLPINTMY(models.AbstractModel):
+    _inherit = "account.edi.xml.ubl_bis3"
+    _name = 'account.edi.xml.pint_my'
+    _description = "Malaysian implementation of Peppol International (PINT) model for Billing"
+    """
+    * PINT Official documentation: https://docs.peppol.eu/poac/pint/pint/
+    * PINT MY Official documentation: https://docs.peppol.eu/poac/my/pint-my
+    """
+
+    def _export_invoice_filename(self, invoice):
+        # EXTENDS account_edi_ubl_cii
+        return f"{invoice.name.replace('/', '_')}_pint_my.xml"
+
+    def _export_invoice_vals(self, invoice):
+        # EXTENDS account_edi_ubl_cii
+        vals = super()._export_invoice_vals(invoice)
+        vals['vals'].update({
+            # see https://docs.peppol.eu/poac/my/pint-my/bis/#profiles
+            'customization_id': 'urn:peppol:pint:billing-1@my-1',
+            'profile_id': 'urn:peppol:bis:billing',
+        })
+        if invoice.currency_id != invoice.company_id.currency_id:
+            # see https://docs.peppol.eu/poac/my/pint-my/bis/#_tax_in_accounting_currency
+            vals['vals']['tax_currency_code'] = invoice.company_id.currency_id.name  # accounting currency
```

Comment:
Aren't we missing this: "the amount of TAX payable in accounting currency is stated in the element Invoice total TAX amount in accounting currency (IBT-111)." ?

Reply:
Nice catch! Yes, should be like in Japan